tests/unittests/graphOptzTest.cpp

```diff
@@ -289,11 +289,25 @@ TEST_F(GraphOptz, cancelTwoTransposes) {
   EXPECT_EQ(F_->getNodes().size(), 2);
 }
 
+TEST_F(GraphOptz, sinkIdentityTranspose) {
+  Node *A = mod_.createVariable(ElemKind::FloatTy, {1, 5, 10, 15}, "input",
+                                VisibilityKind::Public, false);
+  Node *T = F_->createTranspose("transpose", A, {0, 1, 2, 3});
+  Node *K = F_->createRELU("relu", T);
+  F_->createSave("ret", K);
+
+  EXPECT_EQ(F_->getNodes().size(), 3);
+
+  ::glow::optimize(F_, CompilationMode::Infer);
+
+  EXPECT_EQ(F_->getNodes().size(), 2);
```

Comment:
Could you check the actual shape of the resulting graph?
I.e., K's argument is A.

Reply:
fwiw, we do not follow this good practice in the majority of the existing tests, unfortunately :(

Reply:
Done.